pinot-spi/pom.xml

```diff
@@ -179,39 +179,4 @@
       <artifactId>reflections</artifactId>
     </dependency>
   </dependencies>
-  <profiles>
-    <profile>
-      <id>build-shaded-jar</id>
-      <activation>
-        <activeByDefault>false</activeByDefault>
-      </activation>
```

Comment:
After removing this these module level profile the mvn build stopped to publish shaded jars for spi which is required by our build pipeline. We should probably add it back.